lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/domain/Evidence.java

```diff
@@ -13,13 +13,11 @@ public void setTxn(String txn) {
         this.txn = txn;
     }
 
-    public EvidenceType getType() {
+    public String getType() {
         return type;
```

Comment:
simply return VerifiableCredentialConstants.VC_EVIDENCE_TYPE here 
I think the private field can be removed and if necessary an annotation can be added to the getType() method for object mapper  e.g. @JsonGetter("type")